.config/nextest.toml

```diff
@@ -0,0 +1,11 @@
+[test-groups]
+slow = { max-threads = 1 }
```

Comment:
Do we want this to be 1? It seems we want to use as many cores as possible on whatever runner is processing a partition?